src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs

```diff
@@ -31,6 +31,12 @@ public class ReleaseGroupParserFixture : CoreTest
         [TestCase("Black Monday S01 E01-E02 1080p AMZN WEBRip DDP5.1 x264 monkee", null)]
         [TestCase("Series.Title.S01E05.The-Aniversary.WEBDL-1080p.mkv", null)]
         [TestCase("Series.Title.S01E05.The-Aniversary.HDTV-1080p.mkv", null)]
+        [TestCase("Shameless US (2010) S04 (1080p BDRip x265 10bit DTS-HD MA 5 1 - WEM)[TAoE]",null)]
```

Comment:
Only this one fails, the other ones already pass because it'd simply match TrollUHD & RlsGroup already in the original regex. Drop the trailing -RlsGroup in the tests.

Reply:
done.